float32 px_gain
float32 ix_gain
float32 dx_gain
float32 py_gain
float32 iy_gain
float32 dy_gain
float32 pz_gain
float32 iz_gain
float32 dz_gain
float32 pr_gain
float32 ir_gain
float32 dr_gain
float32 pp_gain
float32 ip_gain
float32 dp_gain
float32 pya_gain
float32 iya_gain
float32 dya_gain
float32 awx_gain
float32 awy_gain
float32 awz_gain
float32 awr_gain
float32 awp_gain
float32 awya_gain
float32 integral_lower_limit
float32 integral_upper_limit
int8 window
bool integrator_active
bool controller_active
int8 mode
float32 setpoint1
float32 setpoint2
int8 frequency
float32 osc_point
float32 amplitude
bool ready
